_stbt/core.py

```diff
@@ -490,6 +490,19 @@ def __repr__(self):
                 "<Custom Image>" if isinstance(self.image, numpy.ndarray)
                 else repr(self.image)))
 
+    def __nonzero__(self):
+        return self.match
+
+    def __eq__(self, other):
+        return (self.match == other.match and self.region == other.region and
+                self.image == other.image)
```

Comment:
Similarly, if `MatchResult.image` is a `numpy.ndarray` this will result in a numpy array of bools, which will raise ValueError if you try to use it as a boolean ("The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()").